### PR TITLE
parse yaml files in the readfile

### PR DESCRIPTION
**Pull request by fokion**
Have the ability to parse yaml files as well and give the contentjson

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#65.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#65.1 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

File: executors/readfile/README.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ testcases:
...
     - result.err ShouldBeEmpty
 ```
+```yaml
+name: TestSuite Read Yaml File
+testcases:
+  - name: Read File
+    steps:
+      - type: readfile
+        path: yourfile.yml
+        filetype: yaml
```
Comment (yesnault) [inline executors/readfile/README.md:37]: filetype: no need to add a new attribute filetype. If yml or yaml, we should extract the values, as it's already done with json format.

**Review by yesnault — CHANGES_REQUESTED** [review]

File: executors/readfile/README.md (hunk 1 above)
Comment (yesnault) [inline executors/readfile/README.md:37]:
```suggestion
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#82.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✔ 
* Package-Analyze
  * Docker -   
  * Sonar ✘

**Pull request merged**